internal/gen/gen.go

```diff
@@ -30,6 +30,10 @@ func GoFmt(i Interface) Interface {
 		if err != nil {
 			return nil, err
 		}
-		return format.Source(b)
+		formatted, err := format.Source(b)
+		if err != nil {
+			formatted = b
+		}
+		return formatted, err
```

Comment:
nit: if you accept the change above, this is just:
```suggestion
		return formatted, nil
```